tests/test_build_html.py

```diff
@@ -147,7 +147,7 @@ def check_extra_entries(outdir):
 @pytest.mark.sphinx('html', testroot='warnings')
 def test_html_warnings(app, warning):
     app.build()
-    html_warnings = strip_escseq(warning.getvalue().replace(os.sep, '/'))
+    html_warnings = strip_escseq(re.sub(re.escape(os.sep) + '{1,2}', '/', warning.getvalue()))
```

Comment:
One of the warning messages prints out a repr-string instead of a normal string, which escapes `\`.  Line 33 has the replacement to match it:
```
file u'%(root)s/wrongenc.inc' seems to be wrong, try giving an :encoding: option
```
I forget where in the code this warning is generated, but the `u` prefix on the string is indicative that it's output from a repr-string in the code even though it's not here.